src/main/java/me/totalfreedom/totalfreedommod/command/Command_advertise.java

```diff
@@ -0,0 +1,54 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitScheduler;
+
+@CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
+@CommandParameters(description = "Advertise a message every 10 minutes", usage = "/<command> <message>")
+public class Command_advertise extends FreedomCommand
+{
+    public static ArrayList<Player> cooldown = new ArrayList<Player>();
+    
+    @Override
+    public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        
+        if (cooldown.contains(playerSender))
+        {
+            msg("You must wait 10 minutes to make another advertisement.");
+            return true;
+        }
+        
+        if (args.length < 1)
+        {
+            return false;
+        }
+        
+        String message = StringUtils.join(args, " ", 0, args.length);
+        Bukkit.broadcastMessage(ChatColor.GREEN + "[Advertisement] " + ChatColor.GOLD + sender.getName() + ChatColor.GREEN + "] " + ChatColor.DARK_GREEN + message);
```

Comment:
The string here would make no sense... It would be [Advertisement] wild1145 ] join my server its cool.